source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py

```diff
@@ -132,6 +138,19 @@ def load_managers(self):
         if "startup" in self.event_manager.available_modes:
             self.event_manager.apply(mode="startup")
 
+    def setup_manager_visualizers(self):
```

Comment:
The plots/images from the ManagerLiveVisualizers are made available by the Cfg but then the UI can still activate/deactivate via a checkbox. 

I agree that unifying approach is desired, it think the complication is that sensors (classes) and the mdp terms (functions/classes) / managers (classes) are different from a structural standpoint. I think an argument can be made to embed the ManagerLiveVisualizer as a class member of a Manager rather than there being an EnvLiveVisualizer that handles all of the ManagerLiveVisualizers.

Currently a new plot/image is created for each term of a manager, but the ManagerLiveVisualizer handles the creation. It would complicate the creation of observations  (most of which are tensor functions) to add a LiveVisualizer to them. This why the creation and data collection is handled at the Manager level rather than the mdp term level. 

I will look into the cleaning up activation structure.

Reply:
I would have also expected something more in the direction of @renezurbruegg. As we don't let people choose between different visualization styles (either it is a line plot for everything under 2d or an image for 2d), it could be done by just toggling the expected visualization in the GUI instead of having an own config for it. 

I would not necessarily see the complications as the visualization tool would live in the manager and loop over the same terms and use their output in case the visualization has been enabled for the specific term.

Reply:
After design discussion with @pascal-roth, I will remove the configuration file step of the structure. All control  of the visualizers will fall in the UI checkboxes.